tests/PeriodTest.php

```diff
@@ -109,7 +125,7 @@ public function noOverlappingDates()
              * A          [===]
              * B    [===]
              */
-            [Period::make('2018-02-01', '2018-02-28'), Period::make('2018-01-01', '2018-01-31')],
```

Comment:
Could you undo these styling changes?

Reply:
Rolled back last commit to undo changes.